qchem/requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 scipy
-pennylane>=0.7.0
+pennylane>=0.10.0
 openfermion==0.10.0
 openfermionpsi4==0.4
-pyscf<=1.7.1
+pyscf
```

Comment:
I think there were issues with the latest version  of PySCF (undefined references), that's why it was pinned to version 1.7.1

Reply:
I've reverted it :)